File: Day01-Two-sum/two-sum.py
def twoSums(nums, target):
    seen= {}
    for i , num in enumerate(nums):
        result = target - num
        if result in seen:
            return [seen[result], i]
        seen[num] = i
print(twoSums([2, 7, 11, 15], 9))

File: Day01-Two-sum/two_sum.js
function twoSum(nums,target){
    let seen={};
    for (let i = 0; i<nums.length;i++){
        let complement=target-nums[i];
        if(complement in seen){
            return [seen[complement],i];
        }
        seen[nums[i]]=i
    }
}
console.log(twoSum([2,7,11,15],9))